docs/root/configuration/other_features/dlb.rst

```diff
@@ -22,18 +22,49 @@ An example for Dlb connection balancer configuration is:
 How it works
 ------------
 
-If enabled, the Dlb connection balancer will:
+If enabled, the DLB connection balancer will:
 
-- attach Dlb hardware
+- attach DLB hardware
 - create a queue for balancing
 - create one port to send and one port to receive for each worker thread
 - create one eventfd for each worker thread and attach each eventfd to corresponding customer
-- register each eventfd to corresponding customer and Dlb hardware
+- register each eventfd to corresponding customer and DLB hardware
 
-When new connections come, one worker thread will accept it and send it to Dlb hardware. Dlb hardware
+When new connections come, one worker thread will accept it and send it to DLB hardware. DLB hardware
 does balancing then trigger one worker thread to receive via libevent.
 
-Installing and using Dlb
-------------------------
+Installing DLB
+--------------
 
-For information on how to build/install and use libdlb see `the getting started guide <https://downloadmirror.intel.com/727424/DLB_Driver_User_Guide.pdf>`_.
+You can download the DLB driver release tarball from the `DLB website <https://www.intel.com/content/www/us/en/download/686372/intel-dynamic-load-balancer.html>`_. To install it refer to `the getting started guide <https://downloadmirror.intel.com/727424/DLB_Driver_User_Guide.pdf>`_.
+
+
+Using DLB
+---------
+
+With the following configuration Envoy listens on port 10000 and proxies to an upstream server listening on port 12000.
+
+.. literalinclude:: _include/dlb_example_config.yaml
+ :language: yaml
+ :lines: 1-45
+ :lineno-start: 1
+ :linenos:
+ :caption: :download:`dlb_example_config.yaml <_include/dlb_example_config.yaml>`
+
+Run the upstream service:
+
+.. code-block:: console
+
+ $ docker run -d -p 12000:80 nginx
+
+Run Envoy with DLB connection balancer enabled:
+
+.. code-block:: console
+
+ $ ./envoy --concurrency 2 -c dlb_example_config.yaml
+
+Test:
```

Comment:
not sure we need this section at all - this kinda instruction would be more suited to sandbox/examples - here we are just showing the necessary/minimum configuration to make it work

Reply:
For the "upstream" part and "test" part, I put those here since CI do not have DLB device and cannot run this feature, so I do not put those to sandbox/examples.

My goal is to make an complete example to users, if possible, maybe do not do verify, I can move those to examples, which is more suitable.

Reply:
yeah - this section of docs is meant just as config reference rather than e2e example - given lack of platform support i would be ok with adding a sandbox without the verify.sh in this case

Reply:
I have updated, please review.